### PR TITLE
Cloud: add create swarm doc

### PR DESCRIPTION
**Pull request by sanscontext**
Adding the create-swarm docs so the PR is live and you can merge it whenever.

@borjaburgos @fermayo @pkennedyr 
 
Signed-off-by: LRubin <lrubin@docker.com>

### CODE REVIEW

File: docker-cloud/cloud-swarm/create-cloud-swarm.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+2. Click Swarms in the top navigation.
+3. Click **Create**.
+4. Enter a name for the new swarm.
+5. Select a connected cloud services provider.
+
+    Additional options appear depending on which provider you select.
+
+6. Enter any additional provider-specific information, such as region.
+
+    > **Note**: SSH keys are usually tied to a specific region, so be sure you have appropriate SSH keys available on the region you select.
```
Comment (fermayo) [inline docker-cloud/cloud-swarm/create-cloud-swarm.md]: The list of SSH keys should be automatically filtered by the region selected. Is it not the case?

File: docker-cloud/cloud-swarm/create-cloud-swarm.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+7. Choose how many swarm managers and swarm worker nodes to deploy.
+8. Select the instance sizes for the managers, and for the workers.
+9. Select the SSH key to use to connect to the nodes.
+
+    The list contains any SSH keys that your `dockercloud-user` has access to on your linked cloud services provider.
```
Comment (fermayo) [inline docker-cloud/cloud-swarm/create-cloud-swarm.md]: The user should not select a `dockercloud` SSH key, as the private key is not available to them. We can either remove this or say that they should select a key for which they have the private key locally.

File: docker-cloud/cloud-swarm/create-cloud-swarm.md (hunk 1 above)
Comment (sanscontext) [inline docker-cloud/cloud-swarm/create-cloud-swarm.md]: They are. I'm trying to explain to them that it'll change per region selected. I'll see if I can clarify that.

File: docker-cloud/cloud-swarm/create-cloud-swarm.md (hunk 2 above)
Comment (sanscontext) [inline docker-cloud/cloud-swarm/create-cloud-swarm.md]: Edited. LMK if that fixes it.

File: docker-cloud/cloud-swarm/create-cloud-swarm.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+2. Click Swarms in the top navigation.
+3. Click **Create**.
+4. Enter a name for the new swarm.
+5. Select a connected cloud services provider.
+
+    Additional options appear depending on which provider you select.
+
+6. Enter any additional provider-specific information, such as region.
+
+    > **Note**: The SSH keys that appear in this wizard filtered be the region you select. Ensure that you have appropriate SSH keys available on the region you select.
```
Comment (fermayo) [inline docker-cloud/cloud-swarm/create-cloud-swarm.md]: `The SSH keys that appear in this wizard are filtered by the region you select`

Comment (sanscontext) [inline docker-cloud/cloud-swarm/create-cloud-swarm.md]: Done, sorry!

**Pull request merged**